docs/modules/plugins/pages/plugin-visual.adoc

```diff
@@ -230,6 +230,20 @@ image::example.png[Difference between native/web footer/header image]
 |Default
 |Description
 
+|`mobile.screenshot.android.append-bottom-navigation-bar`
+|`boolean`
+|`true`
+a|Whether to append Android bottom navigation bar (black bar with system buttons on the screenshots below table) or not.
```

Comment:
below table?

Reply:
yep, the screenshots are already there, what's confused you?